cypress/support/custom_commands/login.js

```diff
@@ -14,98 +14,109 @@ const oauth_url =
 const titleOnDashboardPage = '[id="page-title"]'
 
 Cypress.Commands.add('login', (username, environment) => {
-  cy.session([username, environment], () => {
-    const env = Cypress.env()
-    const environmentUpperCased = environment.toUpperCase()
-    const link = Cypress.config('baseUrl', env[environmentUpperCased])
-    cy.log(link)
-    if (environmentUpperCased === 'NBC') {
-      cy.visit('/login')
-      cy.get(nbcLoginWithEmailOptionButton).click()
-    } else if (environmentUpperCased === 'DEFAULT') {
-      cy.visit('/login')
-    } else {
-      cy.visit('/login')
-    }
+  cy.session(
+    [username, environment],
+    () => {
+      const env = Cypress.env()
+      const environmentUpperCased = environment.toUpperCase()
+      const link = Cypress.config('baseUrl', env[environmentUpperCased])
+      cy.log(link)
+      if (environmentUpperCased === 'NBC') {
+        cy.visit('/login')
```

Comment:
Why don't you set the cy.visit('login/') not before the if-else, it is the same for all environments, isn't it?

Reply:
Resolved